internal/pagefile_test.go

```diff
@@ -3,75 +3,262 @@ package internal_test
 import (
 	"bytes"
 	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/utility"
 	"io"
 	"io/ioutil"
 	"log"
 	"os"
 	"testing"
-
-	"github.com/stretchr/testify/assert"
-	"github.com/wal-g/wal-g/internal"
-	"github.com/wal-g/wal-g/utility"
 )
 
 const (
-	pagedFileName        = "../test/testdata/base_paged_file.bin"
-	sampeLSN      uint64 = 0xc6bd4600
+	pagedFileName               = "../test/testdata/base_paged_file.bin"
+	pagedFileSizeInBytes        = 65536
+	pagedFileBlockCount         = pagedFileSizeInBytes / internal.DatabasePageSize
+	sampleLSN            uint64 = 0xc6bd4600
+	smallLSN             uint64 = 0
+	bigLSN                      = sampleLSN * 2
 )
 
-// In this test we use actual postgres paged file which
+var regularTestIncrement = newTestIncrement(sampleLSN)
+var smallLsnTestIncrement = newTestIncrement(smallLSN)
+var bigLsnTestIncrement = newTestIncrement(bigLSN)
+
+// In this test series we use actual postgres paged file which
 // We compute increment with LSN taken from the middle of a file
 // Resulting increment is than applied to copy of the same file partially wiped
 // Then incremented file is binary compared to the origin
 func TestIncrementingFile(t *testing.T) {
-	loclLSN := sampeLSN
-	postgresFileTest(loclLSN, t)
-
+	postgresApplyIncrementTest(regularTestIncrement, t)
 }
 
 // This test covers the case of empty increment
 func TestIncrementingFileBigLSN(t *testing.T) {
-	loclLSN := sampeLSN * 2
-	postgresFileTest(loclLSN, t)
-
+	postgresApplyIncrementTest(bigLsnTestIncrement, t)
 }
 
-// This test converts the case when increment is bigger than original file
+// This test covers the case when increment is bigger than original file
```

Comment:
-> increment is bigger than original file
-> smallLsnTestIncrement
If small Increment is bigger so regular Increment and big Increment is bigger too. 
Looks strange.
Update comment and rename small, regular, and big. It should contain information on how does it related to the file.
If it will be not clear how to put info about why it's different cases into names, then put it to a comment.

Reply:
btw, "original file" -> "the original file"

Reply:
Fixed in ed861e47a77988113cb0d21f7b58b6c483779efe